### PR TITLE
Fix wrong HDD_1 path for zdup-Leap-15.2 job on x86_64

### PR DESCRIPTION
**Pull request by nilxam**

### CODE REVIEW

File: job_groups/opensuse_leap_15.yaml [hunk 1]
```diff
@@ -124,7 +124,9 @@ scenarios:
           machine: 64bit
           settings:
             QEMU_VIRTIO_RNG: "0"
-      - create_hdd_gnome_wicked
+      - create_hdd_gnome_wicked:
+          settings:
+            QEMU_VIRTIO_RNG: "0"
```
Comment (grisu48) [inline job_groups/opensuse_leap_15.yaml:129]: Why is this needed? https://openqa.opensuse.org/tests/3249198 looks fine without the setting.

**Comment by grisu48** [comment]
This PR is referenced in the failing aarch64 test run https://openqa.opensuse.org/tests/3249148#comments. The title suggests however a fix for x86_64, which is not failing: https://openqa.opensuse.org/tests/3249269

I believe or the link is wrong, or the PR aims at doing something else? :thinking:

File: job_groups/opensuse_leap_15.yaml (hunk 1 above)
Comment (nilxam) [reply job_groups/opensuse_leap_15.yaml:129]: it's about other testjob uses wicked asset image, https://openqa.opensuse.org/tests/3253948#step/yast_virtualization/29

**Comment by nilxam** [comment]
> This PR is referenced in the failing aarch64 test run https://openqa.opensuse.org/tests/3249148#comments. The title suggests however a fix for x86_64, which is not failing: https://openqa.opensuse.org/tests/3249269
> 
> I believe or the link is wrong, or the PR aims at doing something else? 🤔

The previous PR https://github.com/os-autoinst/opensuse-jobgroups/pull/311 aim to fix aarch64 but it changed x86_64 as an mistake, thus it breaks x86_64 zdup-leap-15.2 test https://openqa.opensuse.org/tests/3253736, for x86_64 it should boots GM-Updated asset image, and aarch64 has no *updated* image being generated thus GM image is the only available one. This PR corrects that on both aarch64 and x86_64.
